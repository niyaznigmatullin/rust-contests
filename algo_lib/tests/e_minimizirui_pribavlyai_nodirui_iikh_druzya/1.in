7
1 2 3 4 5 6 7
20
4 2 7 10
5 1 6
6 1 6
7 1 6
8 1 6
2 1 6 14
5 2 7
6 2 7
7 2 7
8 2 7
1 2 7 12
5 1 6
6 1 6
7 1 6
8 1 6
3 2 6 15
5 1 7
6 1 7
7 1 7
8 1 7
