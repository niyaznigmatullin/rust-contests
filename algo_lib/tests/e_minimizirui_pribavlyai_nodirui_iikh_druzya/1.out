71
1
16
1
90
14
17
1
74
12
14
2
101
12
15
1
